README.md

```diff
@@ -1,4 +1,4 @@
-# Automated scholarly manuscripts on GitHub
+# Gender and regional biases in scientific coverage revealed through computational analysis of _Nature_ news
```

Comment:
workshopping some things:

* Gender and regional disparities in scientific coverage revealed through computational analysis of _Nature_ news
* Analysis of _Nature_ news reveals gender and regional (biases/disparities) in scientific coverage
* Gender and regional biases (or disparities) in scientific coverage
* Gender and regional (biases/disparities) in _Nature_ news coverage

Reply:
I think "Analysis of _Nature_ news reveals gender and regional disparities in scientific coverage" is both accurate and succinct. Let's go with that for now.

Reply:
updated in https://github.com/greenelab/nature_news_manuscript/pull/2/commits/70798a0775b8c208dbe201ab0cf7b337a59db3c3